ftplugin/php.vim

```diff
@@ -54,5 +54,13 @@ if g:phpactorOmniAutoClassImport == v:true
     autocmd CompleteDone *.php call phpactor#_completeImportClass(v:completed_item)
 endif
 
+let g:phpactorBufferMatcher = get(g:, 'phpactorBufferMatcher', phpactor#project#bufferMatcher#create(
+            \ phpactor#project#projectRepository#create(),
+            \ g:phpactorProjectRootPatterns,
+            \ g:phpactorGlobalRootPatterns,
+            \ g:phpactorInitialCwd
+            \ ))
+
+command PhpactorListProjects echo keys(map(g:phpactorBufferMatcher.getRepository().listProjects(), { key, val -> key}))
```

Comment:
should be in `ftplugin/php/commands.vim` :)

Reply:
Don't forget to make it a buffer command: `:h :command-buffer`